### PR TITLE
build(ffe-icons): create destination directory if needed

### PR DESCRIPTION
**Pull request by henrikhermansen**
This commit makes `ffe-icons` able to create destination directory when
building icons, if it is missing.

### CODE REVIEW

File: packages/ffe-icons/bin/build.js [hunk 1]
```diff
@@ -56,6 +57,12 @@ if (options.projectIcons) {
         });
 }
 
+if (!fs.existsSync(options.dest)) {
```
Comment (wkillerud) [inline packages/ffe-icons/bin/build.js]: `mkdir -p` does nothing if the directory exists, so this check can be removed.

**Review by wkillerud — DISMISSED** [review]

**Comment by henrikhermansen** [comment]
@wkillerud I amended the initial commit and re-added the merge from develop. Your requested changes should be done now :slightly_smiling_face:

File: packages/ffe-icons/bin/build.js [hunk 2]
```diff
@@ -56,6 +57,10 @@ if (options.projectIcons) {
         });
 }
 
+mkdirp.sync(options.dest, error => {
+    if (error) { console.log(error); }
+});
```
Comment (selbekk) [inline packages/ffe-icons/bin/build.js]: As far as I can see from [the docs](https://github.com/substack/node-mkdirp#mkdirpsyncdir-opts), `mkdirp` doesn't accept a function as a second argument. Actually, [looking at the source code](https://github.com/substack/node-mkdirp/blob/master/index.js#L56-L58), it looks like if the second argument isn't an object, it sets the second argument as the `mode` argument - which will potentially bork your access rights.

I suggest that you either just drop the second argument:

```js
mkdirp.sync(options.dest);
```
, or do the same, but add error handling (not sure how real of an issue that will ever be) and a proper error message (and error code) to describe what happened :)

**Review by selbekk — CHANGES_REQUESTED** [review]

File: packages/ffe-icons/bin/build.js (hunk 2 above)
Comment (ivarni) [reply packages/ffe-icons/bin/build.js]: I was actually puzzled by why a synchronous function would accept a callback but just figured the API was wierd :)

Comment (henrikhermansen) [reply packages/ffe-icons/bin/build.js]: Hmm indeed, so it seems.
I actually took this from [a different part of the docs](https://github.com/substack/node-mkdirp#example). Maybe I should file an issue about his example :stuck_out_tongue: 
I'll fix this in a bit!

Comment (ivarni) [reply packages/ffe-icons/bin/build.js]: Yeah, that's the async version. It makes sense for that to take a callback.

Comment (selbekk) [reply packages/ffe-icons/bin/build.js]: @henrikhermansen that part's correct though - if you want to use the asynchronous version, you need to pass a function as a callback. The sync version should probably come with its own example in the docs.

Comment (henrikhermansen) [reply packages/ffe-icons/bin/build.js]: You guys are right! I'm being a total noob right now :man_facepalming:

Comment (henrikhermansen) [reply packages/ffe-icons/bin/build.js]: @selbekk I simply removed the callback as second argument.

**Review by selbekk — APPROVED** [review]

**Pull request merged**